demosplan/DemosPlanCoreBundle/DoctrineMigrations/2023/07/Version20230711125038.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Exception;
+use Doctrine\DBAL\Platforms\MySQLPlatform;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+class Version20230711125038 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'refs 32796 create ProcedureMessage in relation with XBeteiligung ';
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function up(Schema $schema): void
+    {
+        $this->abortIfNotMysql();
+        $this->addSql('ALTER TABLE procedure_message DROP FOREIGN KEY FK_E7F5DA961624BCD2');
+        $this->addSql('DROP INDEX UNIQ_E7F5DA961624BCD2 ON _procedure');
```

Comment:
I get an Error when running this migration... 
the index has to be removed from the correct table
`DROP INDEX UNIQ_E7F5DA961624BCD2 ON procedure_message`